plugins/scaffolder-react/src/next/components/Workflow/Workflow.tsx

```diff
@@ -77,6 +84,64 @@ export const Workflow = (workflowProps: WorkflowProps): JSX.Element | null => {
 
   const { loading, manifest, error } = useTemplateParameterSchema(templateRef);
 
+  const featureFlagKey = 'backstage:featureFlag';
+  const featureFlagApi = useApi(featureFlagsApiRef);
+
+  // return manifest with updates steps
+  const filterOutStepProperties = useCallback(() => {
```

Comment:
Can we move this out into a hook somewhere else other than this component?

Maybe in the `../../hooks` directory.

Something like `useFeatureFlaggedProperties()` or something?